client/src/containers/sign-in/logic/action-types.ts

```diff
@@ -1,15 +1,20 @@
+import { IUser } from 'typings/common/IUser';
 import { ISignInForm } from 'typings/sign-in-form';
-import { WebApi } from 'typings/webapi';
 
 export const SIGN_IN_USER = 'SIGN_IN_USER';
-export const SIGN_IN_USER_SUCCESS = 'SIGN_IN_USER_SUCCESS';
-export const SIGN_IN_USER_ERROR = 'SIGN_IN_USER_ERROR';
+export const SIGN_IN_USER_BY_GITHUB = 'SIGN_IN_USER:GITHUB';
+export const SIGN_IN_USER_SUCCESS = 'SIGN_IN_USER:SUCCESS';
+export const SIGN_IN_USER_ERROR = 'SIGN_IN_USER:ERROR';
```

Comment:
it would be good to use same style for constants' names: either use "_" or ":" separator